### PR TITLE
[18.0][FIX] partner_firstname: Wrong contact type

### PR DESCRIPTION
**Pull request by AEstLo**
There is a typo in the views of contacts

(Migration of https://github.com/OCA/partner-contact/pull/1893)

### CODE REVIEW

**Review by bealdav — APPROVED** [review]
Thanks

File: partner_firstname/__manifest__.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 {
     "name": "Partner first name and last name",
     "summary": "Split first name and last name for non company partners",
-    "version": "18.0.1.0.0",
+    "version": "18.0.1.0.1",
```
Comment (rousseldenis) [inline partner_firstname/__manifest__.py]: If you don't have migration script, don't update this. Thanks

**Review by rousseldenis — CHANGES_REQUESTED** [review]

File: partner_firstname/__manifest__.py (hunk 1 above)
Comment (AEstLo) [reply partner_firstname/__manifest__.py]: Thanks! I thought that I had to update it so odoo.sh knows it has to update this module to apply the changes :)
I'm reverting that

**Review by rousseldenis — APPROVED** [review]

**Comment by houssine78** [comment]
@rousseldenis this one could be merged too :)

**Review by houssine78 — APPROVED** [review]

**Comment by AEstLo** [comment]
@pedrobaeza this is ready to be merged

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge minor

**Pull request merged**